docs/en/patterns/pagination.md

```diff
@@ -5,33 +5,33 @@ table_of_contents: true
 
 ## Pagination
 
-The pagination component should be used to navigate between pages of content.
+<hr>
+
+The pagination component should be used to navigate between pages of content, depending on the length provided, the pagination component will automatically scale.
```

Comment:
I feel like it should be a full stop after "content" because it's a separate piece of information.
`The pagination component should be used to navigate between pages of content. Depending on the length provided, the pagination component will automatically scale.`

Reply:
@Caleb-Ellis reading back I agree. Will update 😉